packages/@glimmer/runtime/lib/scanner.ts

```diff
@@ -58,9 +60,10 @@ export class InlineBlock extends Template {
       builder.bindPositionalArgsForLocals(locals);
     }
 
-    for (let statement of this.statements) {
+    for (let i = 0; i < this.statements.length; i++) {
+      let statement = this.statements[i];
       let refined = SPECIALIZE.specialize(statement, table);
-      STATEMENTS.compile(refined, builder);
```

Comment:
Hmm, no it seems like there is no `b` in this scope, maybe it needs to be reverted?